src/js/types/generic.type.ts

```diff
@@ -1 +1,11 @@
 export type GenericObject = { [key: string]: any };
+
+export type CoreModel = any;
+
+export type PageNode = (Element | ChildNode) & {
+    baseName: string;
+    localName: string;
+    text: string;
+};
+
+export type PlaylistItemType = any;
```

Comment:
Would like to explicitly type PlaylistItem at some point. It's used in a lot of places, could be good to standardize the item types of our events (playlistItem, captionsItem, quality, audioTrack etc.)

Reply:
100% agree! this is a placeholder until we type the actual playlist/item.js file